tests/Volume Testing.postman_collection.json

```diff
@@ -126,7 +126,7 @@
 						"body": {
```

Comment:
Is this name auto-generated from Postman? I changed it to just be lower case "volume" and that's how the Travis tests are setup. We can change the Travis tests or this file name, up to you idk what's best practice for something like this. I would prefer lower case if these aren't autogenerated.

Reply:
They are auto generated which is why I prefer to keep them this way not best practice, but means i can easily export form Postman and paste in file directory after adding tests without having to rename the file. I adjusted corresponding commands in travis setup.